pegasus/sites.v3/code.org/public/pd-workshop-survey/splat.haml

```diff
@@ -4,6 +4,8 @@ title: "Professional Development Workshop Survey"
 :ruby
   dont_cache
   enrollment_code = request.splat_path_info[1..-1]
+  redirect (request.scheme || 'http') + ':' + CDO.studio_url("/pd/workshop_survey/#{enrollment_code}")
```

Comment:
Unfortunately, this isn't working; it's for some reason redirecting me (on localhost) to `http://localhost.code.org:3000/http//localhost-studio.code.org:3000/pd/workshop_survey/6e62a1068c1bfdc6b354`